TypeCobol/Compiler/Sql/Model/SqlExpression.cs

```diff
@@ -68,6 +69,10 @@ public virtual SqlConstantType Type
 
         public Token Literal { get; }
 
+        protected override void DumpContent(TextWriter output, int indentLevel)
+        {
+            DumpProperty(output, nameof(Literal.LiteralValue), Literal.LiteralValue, indentLevel);
```

Comment:
Ok this means that for SqlConstant, we only output the literal value. It's incomplete (no info on the token positioning for example) but it's fine.